beater/query.go

```diff
@@ -127,8 +124,8 @@ func (query *Query) RunQuery(client beat.Client) error {
 		for _, fieldName := range query.config.Fields {
 			wmiObj, err := oleutil.GetProperty(row, fieldName)
 			if err != nil {
-				logp.Err("Unable to get propery by name: %v", err)
-				return err
+				logp.Err("Unable to get property %v: %v", fieldName, err)

```

Comment:
Maybe this is now a warning not an error?
```suggestion
				logp.Warn("Unable to get property %v: %v", fieldName, err)
```